modules/preview-react/status-indicator/lib/StatusIndicatorIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import {createElemPropsHook, createSubcomponent} from '@workday/canvas-kit-react/common';
+
+import {SystemIcon, SystemIconProps} from '@workday/canvas-kit-react/icon';
+import {useStatusIndicatorModel} from './hooks';
+import {colorsMap} from './StatusIndicator';
```

Comment:
This is a circular import: `StatusIndicator.tsx` imports `StatusIndicatorIcon.tsx` and vice versa. You could move the `colorsMap` to this file to avoid the circular dependency. It might work with Webpack, because Webpack knows how to handle circular dependencies, but it may fail in other environments.

Reply:
ah thanks!